### PR TITLE
Misc patches

### PR DESCRIPTION
**Pull request by jduss4**
- updates orchid generator prompt to use `v1` in demo
- standardizes how pagination is being added to search UI
- right aligns pagination
- pulls search results text into own partial and alters wording

Closes https://github.com/CDRH/orchid/issues/39

### CODE REVIEW

File: app/assets/stylesheets/orchid.scss [hunk 1]
```diff
@@ -34,6 +34,10 @@
 // Site Section-specific
 // =====================
 .site_search {
+  .pagination_container {
```
Comment (techgique) [inline app/assets/stylesheets/orchid.scss]: The property should be: `text-align: right;` Space between and ending with a semi-colon.

Also use Sass nesting:
```css
.pagination_container {
  text-align: right;

  .pagination {
    margin: 0;
  }
}
```

Comment (karindalziel) [reply app/assets/stylesheets/orchid.scss]: Do these really need to be nested?

Comment (techgique) [reply app/assets/stylesheets/orchid.scss]: They don't _need_ to be but I prefer it to DRY up naming the parent class. Plus I like how it mimics the parent-child relationship of the elements being styled. It's visually more like how the markup is nested/indented.

Comment (jduss4) [reply app/assets/stylesheets/orchid.scss]: it has been done

**Pull request merged**